templates/index.html

```diff
@@ -5,18 +5,34 @@
     <script src="//ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
   </head>
   <body>
+  <center>
 
     <h1>So it begins...</h1>
 
-    <div id="pile0"></div>
-    <div id="pile1"></div>
-    <div id="pile2"></div>
-    <div id="pile3"></div>
-    <div id="pile4"></div>
+        <h3>
+        Enter Minimum stones per pile <br><input name="desired_min" type="text" size="5"> <br><br>
+        Enter Maximum stones per pile <br><input name="desired_max" type="text" size="5" > <br><br>
+        Enter Number of piles <br><input name="desired_piles" type="text" size="5"> <br><br>
+        </h3>
+
+      <button id="start_button" style="font-size: 100%; height: 60px; width: 80px; background-color: aqua;" class="btn btn-default">Start Game!</button><br><br>
```

Comment:
That button seems redundant with the `New Game` one.

Reply:
I don't think you need this button. We already have the info displayed and the needed buttons available.
Drop this button and you should be good to go.

Reply:
Very well, Removing 🙂